docs/rancher/csi-driver.md

```diff
@@ -40,13 +40,56 @@ When spinning up a Kubernetes cluster using Rancher RKE2 node driver, the Harves
 
 ![select-harvester-cloud-provider](/img/v1.2/rancher/rke2-cloud-provider.png)
 
+You can also deploy Harvester CSI Driver manually if you have already deployed the Harvester node driver manually. Perform the following steps to manually deploy the Harvester CSI Driver. 
+
+1. Generate the addon config.
+
+:::note
+
+The script uses `kubectl` and `jq` to operate the Harvester cluster. The script needs access to the `Harvester Cluster` kubeconfig to work. The `<serviceaccount name>` is usually your guest cluster name, and the `<namespace>` needs to match the namespace of the guest cluster. 
+
+:::
+
+- Install jq:
+  ![](/img/v1.2/rancher/install_jq.png)
+
+- Set up kubectl and kubeconfig:
+  ![](/img/v1.2/rancher/setup_kubectl_and_kubeconfig.png)
+
+You can generate kubeconfig by using the [generate_addon_csi.sh](https://raw.githubusercontent.com/harvester/harvester-csi-driver/master/deploy/generate_addon_csi.sh) script. It is available on the [harvester/harvester-csi-driver](https://github.com/harvester/harvester-csi-driver) repo.
+
+```
+# ./generate_addon_csi.sh <serviceaccount name> <namespace>
+```
+Get the `addon` yaml from the output of the script above.
+
+![](/img/v1.2/rancher/csi_addon_yaml.png)
+
+2. Create the addon config file.
+
+Add the `addon` yaml from the output above.
+```
+# mkdir -p /var/lib/rancher/rke2/etc/config-files
+# vim /var/lib/rancher/rke2/etc/config-files/csi_addon.yaml
+```
```

Comment:
do we need to add the `yaml` file per node or only the control-plane nodes are enough?

Reply:
Yap, we only needed to add the addon yaml file to the control-plane nodes.